crates/import/src/rest.rs

```diff
@@ -0,0 +1,664 @@
+//! Import request collections from VSCode `.rest` files or Jetbrains `.http`
+//! files. VSCode: https://github.com/Huachao/vscode-restclient
+//! Jetbrains: https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html
+
+use anyhow::anyhow;
+use indexmap::IndexMap;
+use itertools::Itertools;
+use serde::de::IgnoredAny;
+use slumber_core::{
+    collection::{
+        Authentication, Chain, ChainId, ChainOutputTrim, ChainSource,
+        Collection, HasId, Method, Profile, ProfileId, Recipe, RecipeBody,
+        RecipeId, RecipeNode, RecipeTree, SelectorMode,
+    },
+    http::content_type::ContentType,
+    template::{Identifier, Template},
+};
+
+use mime::APPLICATION_JSON;
+use reqwest::header::CONTENT_TYPE;
```

Comment:
I don't like fully importing constants or functions because I think it makes them hard to follow. Could you import to the module level and use the qualified identifiers instead? (`mime::APPLICATION_JSON` and `header::CONTENT_TYPE`)